### ISSUE TITLE
Windows installation guide

### ISSUE THREAD

**Issue by Zorakie** [evolus/pencil#170]
* Operating system: Windows 10
* Pencil version: current commit/unknown - cannot open

Hi, I am now using your pencil 2.0.5, which is a great tool, but because of some performance issues, I would like to move to your currtent 3.X.
But missing installation guide. 
Downloaded Electron 1.4.0, drag/dropped /app folder, but getting some error of missing modules(?)
http://screenshot.cz/X4/X4BE2/pencil-error.png

Also got NODEjs+ installed, but cannot see any way, how to link nodejs and electron together to make it work.
So if there would be an installation guide in README.md, it would be great. Thanks :)
(also "uncle google" came with no help :( ... )

**Comment by aschattney** [comment]
you could just download the executable from [http://pencil.evolus.vn/Next.html](http://pencil.evolus.vn/Next.html) ?

**Comment by Zorakie** [comment]
But I would love to have a recent version, it seems the executable build is not updated with every commit

**Comment by aschattney** [comment]
Okay then just follow the section "Setting Up" in the readme:
https://github.com/evolus/pencil#user-content-setting-up

**Comment by Zorakie** [comment]
And that what I want to point to. Some working solution for windows? I have downloaded Atom Electron. The main executable is just a window with an option to drag/drop the folder with app. After dropping, it shows the crazy error on screenshot above. I have also installed nodejs 5+, but its executable is only a command shell. I searched for some config file, where you can direct atom to use libraries from installed nodejs, or something, but I haven't found anything ...

**Comment by aschattney** [comment]
1. `git clone https://github.com/evolus/pencil.git`
2. open the nodejs command prompt
3. navigate to the created directory "pencil" from this command prompt
4. execute `npm install`
5. After installing dependencies finished, execute `npm start` to launch pencil

**Comment by Zorakie** [comment]
worked (almost) well, thaks a lot for your help :) Just npm start is unknown command, but after npm install, drag/drop into Atom electron works fine :) 
(maybe, with this change of point 5), it should be added into readme.md for those as me, who does not know this framework :)  )
but anyway, thank you so much again :) 👍

**Comment by aschattney** [comment]
navigate to the pencil directory in the command prompt and try execute:
`.\node_modules\.bin\electron .\app`
does the app launch now?

**Comment by Zorakie** [comment]
yes, as I said ... after npm install, npm start didnt work, but if I drag/drop into electron, app launches, so no problem, everything works fine ... 

Also - there was one more small difference ... I could not navigate/start npm install from nodejs console, that command I needed to start on regural windows console (just for info)

so if you would be so kind, pease, for other people looking for simmilar solution of the problem, can you add to readme.md installation section those lines as a working solution?

For windows users:
1) git clone https://github.com/evolus/pencil.git
2) install atom electron
3) install nodejs 5+ https://nodejs.org/en/download/
4) open windows command prompt
5) navigate to the created directory "pencil-master/app" from this command prompt
6) execute "npm install"
7) open Atom Electron executable and drop in the APP folder (or you can use console command as atom electron guides in that window
8) Pencil will start 


Thank you :)
